docs/plugins/search/meilisearch.md

```diff
@@ -0,0 +1,108 @@
+---
+icon: https://docsearch.algolia.com/img/favicon.ico
+---
+
+# meilisearch
+
+<NpmBadge package="@vuepress/plugin-meilisearch" />
+
+Integrate [MeiliSearch](https://www.meilisearch.com/) into VuePress, which can provide search to your documentation site.
+
+## Usage
+
+```bash
+npm i -D @vuepress/plugin-meilisearch@next
+```
+
+```ts
+import { MeiliSearchPlugin } from 'vuepress-plugin-meilisearch2'
+
+export default {
+  plugins: [
+    meilisearchPlugin({
+      // Configuration options
+      host: '',
+      apiKey: '',
+      indexUid: '',
+    }),
+  ],
+}
+```
+
```

Comment:
Nope, why I am not doing this is this requires basic guidelines, see the following as examples:

- https://ecosystem.vuejs.press/plugins/blog/comment/waline/
- https://ecosystem.vuejs.press/plugins/blog/comment/giscus/

The guide should be simple as possible, but should not drop any required steps.

If a simple documentation is provided officially, referencing it in the docs is also acceptable, see https://ecosystem.vuejs.press/plugins/blog/comment/artalk/ as example.